cmd/query/Dockerfile

```diff
@@ -8,4 +8,4 @@ COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certifica
 EXPOSE 16686
 COPY query-linux /go/bin/
 ADD jaeger-ui-build /go/jaeger-ui/
-CMD ["/go/bin/query-linux", "--query.static-files=/go/jaeger-ui/"]
+ENTRYPOINT ["/go/bin/query-linux", "--query.static-files=/go/jaeger-ui/"]
```

Comment:
After #918, the `--query.static-files` option is optional and the assets are embedded into the binary, so, remove this from the entrypoint.